spring-cloud-azure-servicebus-stream-binder/src/main/java/com/microsoft/azure/servicebus/stream/binder/properties/ServiceBusExtendedBindingProperties.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.servicebus.stream.binder.properties;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.cloud.stream.binder.ExtendedBindingProperties;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * @author Warren Zhu
+ */
+@Getter
+@Setter
+@ConfigurationProperties("spring.cloud.stream.servicebus")
+public class ServiceBusExtendedBindingProperties
+        implements ExtendedBindingProperties<ServiceBusConsumerProperties, ServiceBusProducerProperties> {
+
+    private Map<String, ServiceBusBindingProperties> bindings = new ConcurrentHashMap<>();
```

Comment:
if expect `bindings` initialize once ? better to add final.

Reply:
Yes